extensions/transactions/create-replace-update-delete/standard/requirements/options/REQ_body.adoc

```diff
@@ -0,0 +1,6 @@
+[[rec_create-replace-delete_options_body]]
+[width="90%",cols="2,6a"]
+|===
+^|*Requirement {counter:req-id}* |*/req/create-replace-delete/options-body*
+^|A |The body of an OPTIONS request SHALL be empty.
```

Comment:
I do not think this should be a requirement or even a recommendation. It is quite common the return the value of the Allow header in text/plain and I do not think this is a bad practice. Can we delete this and maybe add some informative text that makes clear that content in the response may or may not be provided?

Reply:
Meeting 2022-11-21: Remove the requirement and add informative text about the content (or not).

Reply:
Added permission saying request may include a body but format is not defined in this specification and can be safely ignored.